soca/obs/icethk_coperl4.yaml

```diff
@@ -0,0 +1,26 @@
+- obs space:
+    name: icethk_coperl4
+    distribution:
+      name: *obs_distribution
+    obsdatain:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/icethk_coperl4.{{window_begin}}.nc4
+    obsdataout:
+      engine:
+        type: H5File
+        obsfile: $(experiment_dir)/{{current_cycle}}/$(experiment).icethk_coperl4.{{window_begin}}.nc4
+    simulated variables: [sea_ice_category_thickness]
+  obs operator:
```

Comment:
is it "sea_ice_category_thickness" or "sea_ice_thickness"?

Reply:
It is "sea_ice_category_thickness".